src/identity-providers/add/AddMapper.tsx

```diff
@@ -207,6 +219,10 @@ export const AddMapper = () => {
   const isOIDCUsernameTemplateImporter =
     formValues.identityProviderMapper === "oidc-username-idp-mapper";
 
+  const isSocialAttributeImporter = formValues.identityProviderMapper?.includes(
+    "user-attribute-mapper"
+  );
```

Comment:
Since this operation might possibly be intensive to do you could memoize it so it does not get recalculated each time the component is rendered.
```suggestion
  const isSocialAttributeImporter = useMemo(
    () => formValues.identityProviderMapper?.includes("user-attribute-mapper"),
    [formValues.identityProviderMapper]
  );
```